tests/test_presolver.py

```diff
@@ -64,13 +64,14 @@ def test_resolve(presolver):
     presolver.clear()
 
     # We add a path
-    base = os.path.join(os.getenv("ERADIATE_DIR"), "eradiate")
+    base = os.getenv("ERADIATE_DIR")
     presolver.append(base)
+    print(presolver)
```

Comment:
Just out of curiosity, why is this print statement needed here?

Reply:
It's not, it's a mistake. Thanks for catching it!